content/en/docs/concepts/storage/projected-volumes.md

```diff
@@ -138,6 +138,18 @@ Each `podCertificate` projection supports the following configuration fields:
   intermediates).
 * `keyPath` and `certificateChainPath`: Separate paths where Kubelet should
   write *just* the private key or certificate chain.
+* `userAnnotations`: a map that allows you to pass additional information to
+  the signer implementation. It is copied verbatim into the
+  `spec.unverifiedUserAnnotations` field of the [PodCertificateRequest](docs/reference/access-authn-authz/certificate-signing-requests#pod-certificate-requests) objects
+  that Kubelet creates. Entries are subject to the same validation as object
+  metadata annotations, with the addition that all keys must be domain-prefixed.
+  No restrictions are placed on values, except an overall size limitation on the
+  entire field. Other than these basic validations, the API server does not
```

Comment:
I think this warning is probably only necessary on the PodCertificateRequest documentation --- it's a warning to signer implementers, rather than folks using a podCertificate volume.

Reply:
Oh, nvm.  I see Mo had the opposite recommendation above.  I don't particularly mind one way or another.